main/acle.rst

```diff
@@ -404,6 +408,131 @@ Scalable Vector Extensions (SVE)
 ACLE support for SVE is defined in the Arm C Language Extensions for SVE
 document [SVE-ACLE]_ available on the Arm Developer Website.
 
+.. _ssec-NEONSVEBridge:
+
+NEON-SVE Bridge
+---------------
+
+The NEON-SVE Bridge adds intrinsics that enables setting NEON vectors
+into SVE vectors, and getting NEON vectors from SVE vectors. As the first 128
+bit subvector of an SVE vector is physically a NEON register,
```

Comment:
```suggestion
bit subvector of an SVE vector physically overlaps with a NEON register,
```
('is' is a strong word, and the two register classes are distinct)

Reply:
I've removed this sentence entirely after an internal discussion